src/main/java/com/example/Restaurant_Management_System_REST_API/service/ReportService.java

```diff
@@ -42,6 +42,34 @@ public class ReportService {
     private final CellValueSetter<String> stringSetter = new StringValueSetter();
     private final CellValueSetter<Long> longSetter = new LongValueSetter();
     private final CellValueSetter<Double> doubleSetter = new DoubleValueSetter();
+    private static final String ID = "ID";
+    private static final String NAME = "Name";
+    private static final String DESCRIPTION = "Description";
+    private static final String PRICE = "Price";
+    private static final String AMOUNT = "Amount";
+    private static final String SUPPLIER_NAME = "Supplier name";
+    private static final String SUPPLIER_STREET = "Supplier street";
+    private static final String SUPPLIER_HOUSE_NUMBER = "Supplier house number";
+    private static final String SUPPLIER_CITY = "Supplier city";
+    private static final String SUPPLIER_POSTAL_CODE = "Supplier postal code";
+    private static final String SUPPLIER_TELEPHONE_NUMBER = "Supplier telephone number";
+    private static final String CUSTOMER_ID = "Customer ID";
+    private static final String CUSTOMER_NAME = "Customer name";
+    private static final String CUSTOMER_TELEPHONE_NUMBER = "Telephone number";
+    private static final String CREATION_TIME = "Creation time";
+    private static final String RESERVATION_ID = "Reservation ID";
+    private static final String ORDER_TIME = "Order time";
+    private static final String ORDER_STATUS = "Order status";
+    private static final String ORDER_TOTAL_AMOUNT_TO_PAY = "Total amount to pay";
+    private static final String RESTAURANT_ORDER_ID = "Restaurant Order ID";
+    private static final String TABLE_ID = "Table ID";
+    private static final String MENU_RECORD_NAME = "Menu record name";
+    private static final String ACCOUNT_ENABLED = "Account Enabled";
+    private static final String PORTIONS_AMOUNT = "Portions amount";
+    private static final String ROLES = "Roles";
+    private static final String DATE_FORMAT = "yyyy-MM-dd HH:mm:ss";
```

Comment:
This code is better now, as String can be changed in one place and it'll change everywhere. That's basics for example to translate app into another languages. There will be then localization files in separate resource folders and specific files could be used to display different language. Good job!